### DISCUSSION TITLE
Is This Still an Active App?

### DISCUSSION THREAD

**Discussion by kwsim539** [leokhoa/laragon#577] [General]
With 188 open issues and the last release a year old, one would assume this app has been abandoned, causing others I send here to consider using DDEV instead.

**Comment by leokhoa** [comment]
@kwsim539 : Does the current release work for you?

**Reply by RobiNN1** [reply]
You should add a note somewhere that it doesn't need to be updated with every new release of php/apache etc. A lot of people still don't seem to know that they can update everything themselves

**Reply by kwsim539** [reply]
And maybe start resolving the 188 open issues would help too.

**Reply by Pymous** [reply]
@kwsim539 A lot of them need triage, are not Laragon problem, ... the current version works fine ; what does closing those issues fix exactly ?

**Comment by kwsim539** [comment]
> @kwsim539 A lot of them need triage, are not Laragon problem, ... the current version works fine ; what does closing those issues fix exactly ?

If they are specifically not a Laragon issue, then they should be moved to the discussion board and the actual issue closed. The issue is when newbies land here and see 188 issues, their first impressions would be this software has more problems they care to deal with.

**Comment by Pymous** [comment]
Half of the first page of issues are non-issues, and it's visible at a glance ; people wanting to see a lower number to evaluate the actual number of issue shouldn't relate imo.

@leokhoa actually answer most of the "real" issues fairly quickly, and it only took me a minute to figure it out.

Maybe their is a way for us to participate in some form of triage ?

**Comment by rescenic** [comment]
Nah, Laragon 4 is still working for me. No need to migrate.
